auction/src/components/Common/TopBar.js

```diff
@@ -0,0 +1,37 @@
+import React , { useContext } from 'react'
+import { Link } from 'react-router-dom';
+
+import { UserContext } from '../../services/UserContext';
+import { logout } from '../../services/SessionService';
+
+import Social from './Social';
+import Wrapper from './Wrapper';
+
+const TopBar = () => {
+    const { userData, dispatch } = useContext(UserContext);
+    const { loggedIn } = userData;
+    return (
+        <div className='topBar'>
+            <Wrapper flex>
+                <Social/>
+                {
+                    loggedIn ?
+                    <span>
+                        <Link 
+                        to='#'
+                        onClick={e => {
+                            e.preventDefault();
+                            logout(dispatch);
+                        }}>Logout</Link>
```

Comment:
Move to function outside return statement.

Reply:
Done